test/Microsoft.NET.Build.Tests/GivenThatWeWantToBuildADesktopExe.cs

```diff
@@ -173,6 +173,33 @@ public void It_appends_rid_to_outdir_correctly(string identifier, string rid, bo
             }
         }
 
+        [Fact]
+        public void It_builds_multitargeted_with_default_rid()
+        {
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
+            {
+                return;
+            }
+
+            var testAsset = _testAssetsManager
+                .CopyTestAsset("HelloWorld")
+                .WithSource()
+                .WithProjectChanges(project =>
+                {
+                    var ns = project.Root.Name.Namespace;
+                    var propertyGroup = project.Root.Elements(ns + "PropertyGroup").First();
+                    propertyGroup.Element(ns + "TargetFramework").Remove();
+                    propertyGroup.Add(new XElement(ns + "TargetFrameworks", "net46;netcoreapp1.1"));
+                })
+                .Restore();
+
+            var buildCommand = new BuildCommand(Stage0MSBuild, testAsset.TestRoot);
+            buildCommand
+                .Execute()
+                .Should()
+                .Pass();
```

Comment:
Should this be checking that the output is built using the correct RID, as is the case with It_handles_native_depdencies_and_platform_target?